content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -1,475 +1,280 @@
+  Required CN for client certs connecting to the peer endpoint.
+--peer-cert-allowed-hostname ''
+  Allowed TLS hostname for inter peer authentication.
+--peer-auto-tls 'false'
+  Peer TLS using self-generated certificates if --peer-key-file and --peer-cert-file are not provided.
+--self-signed-cert-validity '1'
+  The validity period of the client and peer certificates that are automatically generated by etcd when you specify ClientAutoTLS and PeerAutoTLS, the unit is year, and the default is 1.
+--peer-crl-file ''
+  Path to the peer certificate revocation list file.
+--cipher-suites ''
+  Comma-separated list of supported TLS cipher suites between client/server and peers (empty will be auto-populated by Go).
+--cors '*'
+  Comma-separated whitelist of origins for CORS, or cross-origin resource sharing, (empty or * means allow all).
+--host-whitelist '*'
+  Acceptable hostnames from HTTP client requests, if server is not secure (empty or * means allow all).
+```
+### Auth
+
+```nocode
+--auth-token 'simple'
+  Specify a v3 authentication token type and its options ('simple' or 'jwt').
+--bcrypt-cost 10
+  Specify the cost / strength of the bcrypt algorithm for hashing auth passwords. Valid values are between 4 and 31.
+--auth-token-ttl 300
+  Time (in seconds) of the auth-token-ttl.
+```
+### Profiling and monitoring
+
+```nocode
+--enable-pprof 'false'
+  Enable runtime profiling data via HTTP server. Address is at client URL + "/debug/pprof/"
+--metrics 'basic'
+  Set level of detail for exported metrics, specify 'extensive' to include server side grpc histogram metrics.
+--listen-metrics-urls ''
+  List of URLs to listen on for the metrics and health endpoints.
+```
+### Logging
+
+```nocode
+--logger 'zap'
+  Currently only supports 'zap' for structured logging.
+--log-outputs 'default'
+  Specify 'stdout' or 'stderr' to skip journald logging even when running under systemd, or list of comma separated output targets.
+--log-level 'info'
+  Configures log level. Only supports debug, info, warn, error, panic, or fatal.
+--enable-log-rotation 'false'
+  Enable log rotation of a single log-outputs file target.
+--log-rotation-config-json '{"maxsize": 100, "maxage": 0, "maxbackups": 0, "localtime": false, "compress": false}'
+  Configures log rotation if enabled with a JSON logger config. MaxSize(MB), MaxAge(days,0=no limit), MaxBackups(0=no limit), LocalTime(use computers local time), Compress(gzip)". 
+```
+### Experimental distributed tracing
+
+```nocode
+--experimental-enable-distributed-tracing 'false'
+  Enable experimental distributed tracing.
+--experimental-distributed-tracing-address 'localhost:4317'
+  Distributed tracing collector address.
+--experimental-distributed-tracing-service-name 'etcd'
+  Distributed tracing service name, must be same across all etcd instances.
+--experimental-distributed-tracing-instance-id ''
+  Distributed tracing instance ID, must be unique per each etcd instance.
+```
+### v2 Proxy
```

Comment:
Can we keep it just "Proxy" instead of v2 Proxy.. (similar to how it's originally mentioned ## Proxy flags) Thanks!